docs_src/microservices/support/notifications/Ch-AlertsNotifications.md

```diff
@@ -4,76 +4,114 @@
 
 ## Introduction
 
-When notification to another system or to a person, needs to occur to
-notify of something discovered on the node by another microservice, the
-Alerts and Notifications microservice delivers that information.
-Examples of Alerts and Notifications that other services could need to
-broadcast, include sensor data detected outside of certain parameters
-(usually detected by a Rules Engine service) or system or service
-malfunctions (usually detected by System Management services).
-Terminology
+When another system or a person needs to know that something occurred in EdgeX, the
+alerts and notifications micro service sends that notification.
+Examples of alerts and notifications that other services could broadcast, include the provisioning of a new device, sensor data detected outside of certain parameters
+(usually detected by a device service or rules engine) or system or service
+malfunctions (usually detected by system management services).
+
+### Terminology
 
 **Notifications** are informative, whereas **Alerts** are typically of a
 more important, critical, or urgent nature, possibly requiring immediate
 action.
 
 ![image](EdgeX_SupportingServicesAlertsArchitecture.png)
```

Comment:
Kinda feeling like this image should reflect what we actually have, not what it could be since.

Reply:
I updated the diagram to show "future capability" and changed the non-supported boxes to a different color to highlight this capability is not there today.